src/app/pages/TopicPage/Curation/CurationPromo/index.jsx

```diff
@@ -14,20 +20,55 @@ const CurationPromo = ({
   duration: mediaDuration,
   headingLevel,
 }) => {
+  const { translations } = useContext(ServiceContext);
+
+  const audioTranslation = path(['media', 'audio'], translations);
+  const videoTranslation = path(['media', 'video'], translations);
+  const photoGalleryTranslation = path(['media', 'photogallery'], translations);
+  const durationTranslation = path(['media', 'duration'], translations);
+  const duration = moment.duration(mediaDuration, 'seconds');
+
+  const separator = ',';
+
+  const formattedDuration = formatDuration({ duration, separator });
+  const durationString = `${durationTranslation}, ${formattedDuration}`;
+
   return (
     <Promo>
       <Promo.Image src={imageUrl} alt={imageAlt} lazyLoad={lazy}>
-        <Promo.MediaIcon type={type}>{mediaDuration}</Promo.MediaIcon>
+        <Promo.MediaIcon type={type}>
+          {mediaDuration &&
+          (type === 'video' || type === 'audio' || mediaDuration > 0)
+            ? mediaDuration
+            : ''}
+        </Promo.MediaIcon>
       </Promo.Image>
       <Promo.Heading as={`h${headingLevel}`}>
-        <Promo.A href={link}>{title}</Promo.A>
+        {type !== 'article' ? (
```

Comment:
Similar here, could a `showDuration` variable be used to determine whether or not to render this block?